internal/providers/pluginfw/resources/user/data_users_acc_test.go

```diff
@@ -0,0 +1,62 @@
+package user_test
+
+import (
+	"testing"
+
+	"github.com/databricks/terraform-provider-databricks/internal/acceptance"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const dataSourceTemplate = `
+	resource "databricks_user" "user1" {
+		user_name = "testuser1@databricks.com"
```

Comment:
```suggestion
		user_name = "user-datasource-{var.STICKY_RANDOM}-1@databricks.com"
```
Tests run in a single testing environment. Tests use isolated state files for resources created during the tests, but the resources are created in a single environment. Two tests running in parallel may create and delete the same user. Use `{var.STICKY_RANDOM}` when creating resources that need unique names or other attributes to ensure they don't clash with one another.

Reply:
Gotcha, thanks! 👍